plugin/node_resolve.js

```diff
@@ -15,6 +15,7 @@
   function findDeclaredDeps() {}
 
   var resolveToFile
+  var projectPaths = [];
```

Comment:
Put this in the `Modules` object, not as a file-global variable. (You could have multiple Tern servers active in a process, with different configurations.)